docs/docs/learn/reference/audits/index.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: Audits
 
 ## BOB Mainnet
 
-In Phase 1 of our [roadmap](/learn/introduction/roadmap.md), "Optimistic BOB" is an OP-stack rollup on Ethereum. We inherit the hard work of the Optimism Collective, including the many [security reviews](https://github.com/ethereum-optimism/optimism/tree/v1.1.4/technical-documents/security-reviews) they have secured for our infrastructure.
+In Phase 1 of our [roadmap](/learn/introduction/roadmap), "Optimistic BOB" is an OP-stack rollup on Ethereum. We inherit the hard work of the Optimism Collective, including the many [security reviews](https://github.com/ethereum-optimism/optimism/tree/v1.1.4/technical-documents/security-reviews) they have secured for our infrastructure.
 
```

Comment:
Should add the audits we did for the upgradable ERC20 contracts (https://github.com/bob-collective/optimism/pull/2) and USDC (https://github.com/bob-collective/optimism/pull/1). @gregdhill or @sander2 should have these.

Reply:
@gregdhill, any luck finding these two audits yet?

Reply:
We don't need to block this PR on these audit reports. Let's just add a issue to add them.

Reply:
Noted. I'll include them in a following PR.